.github/pull_request_template.md

```diff
@@ -19,14 +19,13 @@ This PR introduces this capability to make the project better in this and that.
 not fully covered by unit tests or manual testing can be complicated. -->
 
 ### Checklist
-
 <!-- Put an 'x' in all the boxes that apply -->
-
-- [ ] I submit my changes into the `develop` branch
-- [ ] I have added description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md)
-- [ ] I have updated the [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) accordingly
-- [ ] I have added tests to cover my changes
-- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)
+- [ ] I have tested the changes. It works.​
+- [ ] I have tested the features that may be affected by this changes.​
```

Comment:
Can't we remove these 2? Because they are automatically verified by unit / integration tests.

Reply:
Okay